### PR TITLE
profile-modal

### PR DESCRIPTION
**Pull request by yashkapoor8710**
Simulator Screen Shot - iPhone 11 - 2020-02-21 at 16 13 34

### CODE REVIEW

File: components/Chat/ChatUserImage.tsx [hunk 1]
```diff
@@ -1,21 +1,59 @@
-import React from 'react';
-import { Image, ImageStyle, ImageSourcePropType, View, ViewStyle } from 'react-native';
+import React, { useState } from 'react';
+import { Modal, Image, ImageStyle, ImageSourcePropType, View, ViewStyle, StyleSheet, TouchableOpacity, Dimensions } from 'react-native';
 
 interface Props {
   containerStyle?: ViewStyle;
   source: ImageSourcePropType;
-  style: ImageStyle;
+  styleImage: ImageStyle;
```
Comment (manojsinghnegiwd) [inline components/Chat/ChatUserImage.tsx]: change to `imageStyle`

File: components/Chat/ChatUserImage.tsx [hunk 2]
```diff
@@ -1,21 +1,59 @@
...
 };
 
-const ChatUserImage: React.FunctionComponent<Props> = ({ source, containerStyle, style }: Props) => {
+const ChatUserImage: React.FunctionComponent<Props> = ({ source, containerStyle, styleImage }: Props) => {
+
+  const [Modals, setModalVisible] = useState<boolean>(false);
+
+  const handleModal = (visible: boolean): void => {
```
Comment (manojsinghnegiwd) [inline components/Chat/ChatUserImage.tsx]: Remove this.

File: components/Chat/ChatUserImage.tsx [hunk 3]
```diff
@@ -1,21 +1,59 @@
...
+}
+
+const style: Style = StyleSheet.create<Style>({
+  modalContainer: {
+      flexDirection: 'column',
+      justifyContent: "center",
+      alignItems: 'center',
+      width: Dimensions.get('window').width,
+      height: Dimensions.get('window').height,
+      backgroundColor: 'rgba(0,0,0,0.5)'
```
Comment (manojsinghnegiwd) [inline components/Chat/ChatUserImage.tsx]: Move it to theme

File: components/Chat/ChatUserImage.tsx (hunk 1 above)
Comment (yashkapoor8710) [reply components/Chat/ChatUserImage.tsx]: I have changed it

File: components/Chat/ChatUserImage.tsx (hunk 2 above)
Comment (yashkapoor8710) [reply components/Chat/ChatUserImage.tsx]: I have removed this

File: components/Chat/ChatUserImage.tsx (hunk 3 above)
Comment (yashkapoor8710) [reply components/Chat/ChatUserImage.tsx]: I have move it to theme

File: components/Chat/ChatUserImage.tsx [hunk 4]
```diff
@@ -1,21 +1,59 @@
...
+}
+
+const style: Style = StyleSheet.create<Style>({
+  modalContainer: {
+      flexDirection: 'column',
+      justifyContent: "center",
+      alignItems: 'center',
+      width: Dimensions.get('window').width,
+      height: Dimensions.get('window').height,
+  },
```
Comment (manojsinghnegiwd) [inline components/Chat/ChatUserImage.tsx]: You can do something like this.

```
const { height, width } = Dimensions.get('window')
```

Comment (yashkapoor8710) [reply components/Chat/ChatUserImage.tsx]: okay I have used this

**Pull request merged**